.env.example

```diff
@@ -35,6 +35,8 @@ CONTENT_SYNC_THEME_PIPELINE=content_sync.pipelines.concourse.ThemeAssetsPipeline
 # Values used in pipeline definitions
 OCW_STUDIO_BASE_URL=http://MYNAME.ocw-studio.odl.local:8043/
 SEARCH_API_URL=https://discussions-rc.odl.mit.edu/api/v0/search/
+COURSE_SEARCH_API_URL=https://mit-open-rc.odl.mit.edu//api/v1/learning_resources_search/
```

Comment:
This should be `COURSE_SEARCH_API_URL=https://mit-open-rc.odl.mit.edu/api/v1/learning_resources_search/` without the double slash

Reply:
oops